lib/uo_cli.rb

```diff
@@ -198,13 +198,13 @@ def opt_visualize
         @subopts = Optimist.options do
           banner "\nURBANopt #{cmd}:\n \n"
 
-          opt :scenarios, "\nVisualize results for all scenarios\n" \
+          opt :feature, "\nVisualize results for all scenarios for a feature file\n" \
             "Provide the FeatureFile whose scenario results you want to visualize\n" \
-            'Example: uo visualize --scenarios example_project.json', type: String
+            'Example: uo visualize --feature example_project.json', default: 'example_project.json'
 
-          opt :features, "\nVisualize results for all features in a scenario\n" \
+          opt :scenario, "\nVisualize results for all features in a scenario\n" \
             "Provide the Scenario whose feature results you want to visualize\n" \
```

Comment:
"Provide the scenario CSV file to visualize each feature in the scenario."

also i'm not sure defaults make sense here...

Reply:
and can you also hard code the "short" option names here in case we add things in the future?

Reply:
@kflemin I'm not sure I followed hard coding the short option names

Reply:
@tanushree04 https://github.com/urbanopt/urbanopt-cli/blob/develop/lib/uo_cli.rb#L102